department-of-veterans-affairs/caseflow
# Remove ability to reassign an AMA judge task from the 'Assign to attorney' page or modal

Resolves #11761

### Description
Essentially reverting #10006. When on the 'Assign cases' page for a judge, or within the "Assign to attorney" modal, only attorney tasks can be created for AMA cases. Reassigning an AMA judge task  is already available via the "Re-assign to judge" dropdown on the case details page.

### Acceptance Criteria
- [ ] `JudgeAssignTask`s cannot be reassigned to a judge from the 'Assign cases' page
- [ ] `AttorneyTask`s can be assigned to an acting judge from the 'Assign cases' page
- [ ] `JudgeAssignTask`s cannot be reassigned to a judge from the 'Assign to attorney' action
- [ ] `AttorneyTask`s can be assigned to an acting judge from the 'Assign to attorney' action
- [ ] Errors direct users on how to properly reassign cases to a judge

### Testing Plan
1. Sign in as a judge (BVAAABSHIRE) and navigate to the Assign cases page
2. Attempt to assign an AMA case to a judge
3. Confirm errors
4. Create an acting judge
```ruby
judge = User.find_by(css_id: "BVAAABSHIRE")
atty = FactoryBot.create(:user, css_id: "BVALODELL", full_name: "Lorena Odell")
vacols_atty = FactoryBot.create(:staff, :attorney_judge_role, user: atty)
OrganizationsUser.add_user_to_organization(atty, JudgeTeam.for_judge(judge))
```
5. Attempt to assign a case to Lorena Odell
6. Confirm this works
7. Repeat 2 - 6 from the "Assign to attorney" drop down action on the case details screen
